cogs/tmpc.py

```diff
@@ -297,50 +307,12 @@ async def token(self, ctx: Context, mode: str):
             message: Message = await ctx.send(embed=embed)
             await ctx.message.delete()
             await message.add_reaction(emoji="🔓")
-            new_document = {
-                DBKeyWrapperEnum.OWNER.value: document.owner_id,
-                DBKeyWrapperEnum.CHAT.value: document.channel_id,
-                DBKeyWrapperEnum.VOICE.value: document.voice_id,
-                DBKeyWrapperEnum.TOKEN.value: document.token,
-                DBKeyWrapperEnum.DELETE_AT.value: document.deleteAt,
-                DBKeyWrapperEnum.MESSAGES.value: document.messages + [message]
-            }
-            new_document[DBKeyWrapperEnum.MESSAGES.value] = [(message.channel.id, message.id) for message in
-                                                             new_document[DBKeyWrapperEnum.MESSAGES.value]]
-            await self.study_db.update_one(document.document, new_document)
-            return
-
-        if mode.lower() == "show":
-            document = await self.check_tmpc_channel(ctx.author, ctx.channel.id, everyone=True)
-            embed: Embed = Embed(title="Token",
-                                 description=f"`!tmpc join {document.token}`")
```

Comment:
why no token show anymore?

Reply:
Token place does the same but better (reaction and self deletion).

Reply:
I still think token show should stay for ease of use if the token is far up in the temp channel and people just want the token. Because else there would be this big message not necessary

Reply:
Okay but `token show` should behave equal (self deletion).
Will change that in own PR.

Reply:
No, `token show` is only used in the tmp chat so leave it when asked, no needed to make it self delete

Reply:
Okay.